Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackConfiguration.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import UIKit
+
+/**
+ * The settings to use for how the user feedback form is presented, what data is required and how
+ * it's submitted, and some auxiliary hooks to customize the workflow.
+ */
+@objcMembers public class SentryUserFeedbackConfiguration: NSObject {
+    /**
+     * Configuration settings specific to the managed widget that displays the UI form.
+     * - note: Default: `nil` to use the default widget settings.
+     */
+    public var configureWidget: ((SentryUserFeedbackWidgetConfiguration) -> Void)?
+    
+    /**
+     * Whether users should be able to drag the widget to new locations.
+     * - note: Default: `false`
+     */
+    public var draggable: Bool = false
+    
+    /**
+     * Use a shake gesture to display the form.
+     * - note: Default: `false`
+     * - note: Setting this to true does not disable the widget. In order to do so, you must set `SentryUserFeedbackWidgetConfiguration.autoInject` to `false` using the `SentryUserFeedbackConfiguration.configureWidget` config builder.
+     */
+    public var useShakeGesture: Bool = false
```

Comment:
`m`: We don't have these two features on the web, I believe. While they add value, we could skip them in the first iteration if we want to narrow the scope a bit.

Reply:
You are correct. I think the shake gesture would be trivial to add, but we can defer the widget draggability feature.